### PR TITLE
Prepare CumulusCI 3.48.0

### PR DESCRIPTION
**Pull request by codexmarcel**
# Critical Changes

# Changes
This PR is part of the release process for `CumulusCI`.

# Issues Closed

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,8 +2,22 @@
 History
 =======
 
+v3.48.0 (2021-11-11)
+--------------------
+Changes
```
Comment (davisagli) [inline HISTORY.rst:7]: We should include a Critical Changes section warning of the upcoming dropping of support for Python 3.6 and 3.7

File: HISTORY.rst [hunk 2]
```diff
@@ -2,8 +2,22 @@
...
+
+- We added a new command, ``cci plan info``. Similar to ``cci task info``, this command displays detailed information about a MetaDeploy ``plan``, and includes a ``--messages`` option to display user-facing text.
```
Comment (davisagli) [inline HISTORY.rst:9]: PR#?

File: HISTORY.rst [hunk 3]
```diff
@@ -2,8 +2,22 @@
...
+- Improved logging to show reduced timestamps, and provide some syntax highlighting of output. (#2941)
+- `Snowfakery 2.2 <https://github.com/SFDO-Tooling/Snowfakery/releases/tag/v2.2>` is now included with CumulusCI. New features include unique IDs and Numeric Counters. (#2962)
```
Comment (davisagli) [inline HISTORY.rst]:
```suggestion
- `Snowfakery 2.2 <https://github.com/SFDO-Tooling/Snowfakery/releases/tag/v2.2>`__ is now included with CumulusCI. New features include unique IDs and Numeric Counters. (#2962)
```

**Comment by davisagli** [comment]
For future releases we should figure out a good process for @pconrad-sfdo to be involved in writing the release notes. Your feedback is welcome here.

**Comment by pconrad-sfdo** [comment]
> For future releases we should figure out a good process for @pconrad-sfdo to be involved in writing the release notes. Your feedback is welcome here.

I don't know the whole pipeline yet, of course, but I'm happy to take a look at any PRs for grammar/tone etc.

File: HISTORY.rst (hunk 1 above)
Comment (Br4nd0R) [reply HISTORY.rst:7]: I think something like:

> CumulusCI will be dropping support for Python 3.6 and 3.7 in the next couple of releases. Please ensure you are running Python>3.8 to ensure you won't be impacted by this change.

**Review by Br4nd0R — DISMISSED** [review]

**Review by Br4nd0R — APPROVED** [review]

File: HISTORY.rst [hunk 4]
```diff
@@ -4,9 +4,12 @@ History
 
 v3.48.0 (2021-11-11)
 --------------------
+Critical Changes
+- CumulusCI will be dropping support for Python 3.6 and 3.7 within the next few releases. Please ensure your environment may run Python 3.8 or greater.
```
Comment (pconrad-sfdo) [inline HISTORY.rst]: How about:

```
- CumulusCI will be dropping support for Python 3.6 and 3.7 within the next few releases. Please ensure you're running Python 3.8 or above.
```

**Review by Br4nd0R — APPROVED** [review]

**Pull request merged**